ktxspec.adoc

```diff
@@ -1148,11 +1148,14 @@ have a sample with a channelType that indicates it is alpha.
 ===== DFD for Supercompressed Data
 When `<<_supercompressionscheme,supercompressionScheme>>` is not 0
 the _dfDescriptorBlock_ must preserve the `colorModel`, `transferFunction`,
-`colorPrimaries`, `flags` and `texelBlockDimension[0-3]` of the
-pre-deflation images along with each sample's `channelType`,
-`qualifiers`, `bitlength`, `bitOffset`, `sampleLower` and `sampleUpper`.
-`bytesPlane[0-7]` must be set to 0 to indicate an _unsized format_,
-as described in Section 5.19 _Unsized Formats_ of <<KDF13>>.
+`colorPrimaries`, `flags`, `texelBlockDimension[0-3]` and
+`bytesPlane[0-7]` of the pre-deflation images along with each
+sample's `channelType`, `qualifiers`, `bitlength`, `bitOffset`,
+`sampleLower`, and `sampleUpper`.
+
+BasisLZ supercompression works only on ETC1S blocks so
+`texelBlockDimension[01]` must be 3 and `bytesPlane0` must be
+8 and, if there is an alpha plane, `bytesPlane1` must also be 8.
```

Comment:
Nit: this statement does not make it clear that these requirements apply only to BasisLZ. Consider ending it with " when BasisLZ is used" (or rephrasing).